daprdocs/content/en/reference/components-reference/supported-pubsub/setup-rabbitmq.md

```diff
@@ -412,6 +412,24 @@ client.PublishEvent(ctx, PUBSUB_NAME, TOPIC_NAME, []byte(strconv.Itoa(orderId)),
 
 {{< /tabs >}}
 
+## Use quorum queues
+
+By default, Dapr creates `classic` queues. To create `quorum` queues, add the following metadata to your subscription:
```

Comment:
```suggestion
By default, Dapr creates `classic` queues. To create `quorum` queues, add the following metadata to your pub/sub [subscription]({{< ref subscription-schema.md >}})
```

Reply:
@aaguilartablada - Got it. Thanks for the clarification. I closed the other PR (https://github.com/dapr/docs/pull/3193_ and looked at the code in yours to see this change. Suggest adding this link to the subscription format